app/views/webpages/index.html.erb

```diff
@@ -2,61 +2,70 @@
   <%= render "carousel" %>
 </div>
 
-<div style="background-color:#f3f3f3;">
+<div id="home-background">
 
-<div class="container" style="">
+<div class="container homepage">
 
 <div class="row">
   <div class="col col-3 mt-4" id="left-home-menu">
-    <ul class="list-unstyled" style="margin-bottom:0;color:#A50030">
-      <li>Resources for Authors</li>
-      <li>Resources for Booksellers</li>
-      <li>Conferences</li>
-      <li>Special Offers</li>
-      <li>Books for Course Adoption</li>
-      <li>Read Our Blog</li>
-      <li style="margin-bottom:0;">Donate to the Press</li>
+    <ul class="list-unstyled">
+      <li><%= link_to "Resources for Authors", "#" %></li>
+      <li><%= link_to "Resources for Booksellers", "#" %></li>
+      <li><%= link_to "Conferences", "#" %></li>
+      <li><%= link_to "Special Offers", "#" %></li>
+      <li><%= link_to "Books for Course Adoption", "#" %></li>
+      <li><%= link_to "Read Our Blog", "#" %></li>
+      <li><%= link_to "Donate to the Press", "#" %></li>
     </ul>
   </div>
-  <div class="col col-9 mt-4 pt-2 pb-1" style="background-color: white;">
-    <h1 style="text-decoration:underline;margin-left: 35px">New & Forthcoming</h1>
+
+  <div class="col col-9 mt-4 pt-2 pb-1" id="new-forthcoming">
+    <h1>New & Forthcoming</h1>
     <div class="row pt-3">
       <div class="col-4 text-center">
-        <%= image_pack_tag "Giving_Back.jpg", style: "height:300px;max-width:200px;" %>
-        <p style="margin-top: 7px;color:#A50030"><strong>Giving Back</strong></p>
+      <%= link_to "#" do %>
+        <%= image_pack_tag "Giving_Back.jpg" %>
+        <p><strong>Giving Back</strong></p>
+      <% end %>
       </div>
       <div class="col-4 text-center">
-      <%= image_pack_tag "Illegal_Immigrants_Model_Minorities.jpg", style: "height:300px;max-width:200px;" %>
-        <p style="margin-top: 7px;color:#A50030"><strong>Illegal Immigrants Model Minorities</strong></p>
+      <%= link_to "#" do %>
+      <%= image_pack_tag "Illegal_Immigrants_Model_Minorities.jpg" %>
+        <p><strong>Illegal Immigrants Model Minorities</strong></p>
+      <% end %>
       </div>
       <div class="col-4 text-center">
-      <%= image_pack_tag "Women_s_Empowerment.jpg", style: "height:300px;max-width:200px;" %>
-        <p style="margin-top: 7px;color:#A50030"><strong>Women's Empowerment and Disempowerment in Brazil</strong></p>
+      <%= link_to "#" do %>
+      <%= image_pack_tag "Women_s_Empowerment.jpg" %>
```

Comment:
@cdoyle-temple Do these images need alt tags? Or are you adding that later?

Reply:
These images are treated as decoration as they supply no more information than the adjacent title text on the page. Screen readers skip such images with null alt values.